WordPress/Classes/ViewRelated/Stats/Extensions/StatsViewController+JetpackSettings.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import UIKit
+
+extension StatsViewController {
+
+    @objc func activateStatsModule(success: @escaping () -> Void, failure: @escaping (Error?) -> Void) {
```

Comment:
I have some worries about extending the `StatsViewController` directly and creating an instance of the service without dependency injection.

However, given this is an edge case and that a refactor might increase the cost of this task I think we're ok for now. But if we keep working on the Stats we should really think about a refactor/modernization on this VC. :)